interface/procedure_tools/labcorp/ereq_form.php

```diff
@@ -0,0 +1,775 @@
+        $pdfContent .= '</tr>';
+        $pdfContent .= '<tr><td class="width50" style="width:50%;">Date</td></tr>';
+        $pdfContent .= '</table>';
+        $pdfContent .= '</td>';
+        $pdfContent .= '</tr>';
+        $pdfContent .= '</table>';
+        $pdfContent .= '<table>';
+        $pdfContent .= '<tr>';
+        $pdfContent .= '<td class="width50">';
+        $pdfContent .= '<table style="width:250px;">';
+        $pdfContent .= '<tr>';
+        $pdfContent .= '<td class="width50" style="width:50%;height: 40px;padding-top:40px;border-bottom:1px solid #000;">&nbsp;</td>';
+        $pdfContent .= '</tr>';
+        $pdfContent .= '<tr><td class="width50" style="width:50%;">Patient Signature</td></tr>';
+        $pdfContent .= '</table>';
+        $pdfContent .= '</td>';
+        $pdfContent .= '<td class="width50">';
+        $pdfContent .= '<table style="width:150px;">';
+        $pdfContent .= '<tr>';
+        $pdfContent .= '<td class="width50" style="width:50%;height: 40px;padding-top:40px;border-bottom:1px solid #000;">&nbsp;</td>';
+        $pdfContent .= '</tr>';
+        $pdfContent .= '<tr><td class="width50" style="width:50%;">Date</td></tr>';
+        $pdfContent .= '</table>';
+        $pdfContent .= '</td>';
+        $pdfContent .= '</tr>';
+        $pdfContent .= '</table>';
+        $pdfContent .= '</td>';
+        $pdfContent .= '</tr>';
+        $pdfContent .= '</tbody>';
+        $pdfContent .= '</table>';
+        $pdfContent .= '</div>';
+        $pdfContent .= '</body>';
+        $pdfContent .= '</html>';
+        $mpdf->WriteHTML($styleSheet, 1);
+        $mpdf->WriteHTML($pdfContent, 2);
+
+        $unique = date('y-m-d-His', time());
+        $filename = "ereq_" . $unique . "_order_" . $form_id . ".pdf";
+
+//-----------patch in exclude dodoc
+        if ($_REQUEST['debug'] || !$doDoc) {
+            $mpdf->Output($filename, "I");
+            return;
+        }
+        // trickery to create and then remove the temporary file
+        //  (so can then create it anew via mPDF)
+        $tmpFile = tempnam($GLOBALS['temporary_files_dir'], "ereq");
+        unlink($tmpFile);
+        $mpdf->Output($tmpFile, "F");
+        $mpdfData = file_get_contents($tmpFile);
+        unlink($tmpFile);
+
+// register the new document
+        $category = sqlQuery("SELECT id FROM categories WHERE name LIKE ?", array("LabCorp"));
+        if (!$category['id']) {
+            $category = sqlQuery("SELECT id FROM categories WHERE name LIKE ?", array('Lab Report'));
+        }
+        $DOCUMENT_CATEGORY = $category['id'];
+
+        $d = new Document();
+        $docid = $d->createDocument($pid, $DOCUMENT_CATEGORY, $filename, "application/pdf", $mpdfData);
+        $unique = date('y-m-d-H:i:s', time());
+        $documentationOf = "$unique";
+        sqlStatement(
+            "UPDATE documents SET documentationOf = ? WHERE id = ?",
+            array($documentationOf, $docid)
```

Comment:
$d->id also need the records list_id for reference to procedure.
Once I fixed this, it works great so thank you.
I plan to make this req universal for any lab so that'd be cool.